src/runtime/pythonengine.cs

```diff
@@ -146,6 +146,24 @@ public static int RunSimpleString(string code)
             return Runtime.PyRun_SimpleString(code);
         }
 
+        static bool libraryLoaded = false;
+        public static void InitializeLibrary()
+        {
+            if (!libraryLoaded)
+            {
+#if MONO_OSX
+                var os = Python.Runtime.Platform.OperatingSystemType.Darwin;
+#elif MONO_LINUX
+                var os = Python.Runtime.Platform.OperatingSystemType.Linux;
+#else
+                var os = Python.Runtime.Platform.OperatingSystemType.Windows;
+#endif
```

Comment:
We are trying to get rid of compile-time checks for the OS. Please use equivalent runtime information, there should already be code in place on master to do this.